src/pages/api/sendEmail.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable prettier/prettier */
+import { NextApiRequest, NextApiResponse } from 'next';
+
+const nodemailer = require('nodemailer');
+
+type Data = {
+  message?: string;
+  error?: string;
+  details?: string;
+};
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<Data>
+) {
+  const { to, subject, text } = req.body;
+
+  // Create a transporter object
+  try {
+    const transporter = nodemailer.createTransport({
+      service: 'gmail',
+      auth: {
+        user: ['kellypham@uwblueprint.org'],
```

Comment:
I would put these variables in the .env file instead of hardcoding them.

Reply:
Please contact me to put the email / password in the hashicorp vault. I feel like we should probably use the sistema email instead of kelly's so let me know when you would like to set that up!

Reply:
done